Arguments: 
  /usr/local/bin/node /usr/local/Cellar/yarn/1.5.1_1/libexec/bin/yarn.js dev

PATH: 
  /Users/vdhulappanavar/anaconda3/bin:/usr/local/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin

Yarn version: 
  1.5.1

Node version: 
  8.9.4

Platform: 
  darwin x64

npm manifest: 
  {
    "name": "react-socket-app",
    "version": "1.0.0",
    "description": "",
    "main": "server.js",
    "scripts": {
      
      "client": "cd client && yarn start",
      "server": "nodemon server.js",
      "dev": "concurrently --kill-others-on-fail \"yarn server\" \"yarn client\""
    },
    "repository": {
      "type": "git",
      "url": "git+https://github.com/vdhulappanavar/Stackroute_socketio.git"
    },
    "keywords": [],
    "author": "",
    "license": "ISC",
    "bugs": {
      "url": "https://github.com/vdhulappanavar/Stackroute_socketio/issues"
    },
    "homepage": "https://github.com/vdhulappanavar/Stackroute_socketio#readme",
    "dependencies": {
      "express": "^4.16.3",
      "socket.io": "^2.0.4"
    },
    "devDependencies": {
      "concurrently": "^3.5.1"
    }
  }

yarn manifest: 
  No manifest

Lockfile: 
  No lockfile

Trace: 
  Error: Command failed.
  Exit code: 1
  Command: sh
  Arguments: -c concurrently --kill-others-on-fail "yarn server" "yarn client"
  Directory: /Users/vdhulappanavar/Documents/StackRouteLearning/react-socket-app
  Output:
  
      at ProcessTermError.MessageError (/usr/local/Cellar/yarn/1.5.1_1/libexec/lib/cli.js:186:110)
      at new ProcessTermError (/usr/local/Cellar/yarn/1.5.1_1/libexec/lib/cli.js:226:113)
      at ChildProcess.<anonymous> (/usr/local/Cellar/yarn/1.5.1_1/libexec/lib/cli.js:30281:17)
      at emitTwo (events.js:126:13)
      at ChildProcess.emit (events.js:214:7)
      at maybeClose (internal/child_process.js:925:16)
      at Process.ChildProcess._handle.onexit (internal/child_process.js:209:5)
